templates/spec/requests/carts_spec.rb

```diff
@@ -55,32 +44,23 @@
       it "renders the edit view (on failure)" do
         # email validation is only after address state
         order.update_column(:state, "delivery")
-        put order_path(order.number), params: { order: { email: "" } }
+        patch cart_path, params: { order: { email: "" } }
         expect(response).to render_template :edit
       end
 
       it "redirects to cart path (on success)" do
-        put order_path(order.number), params: { order: { email: 'test@email.com' } }
-        expect(response).to redirect_to(cart_path)
+        patch cart_path, params: { order: { email: 'test@email.com' } }
+        expect(response).to redirect_to(edit_cart_path)
       end
 
       it "advances the order if :checkout button is pressed" do
         expect do
-          put order_path(order.number), params: { checkout: true }
+          patch cart_path, params: { checkout: true }
         end.to change { order.reload.state }.from('cart').to('address')
 
         expect(response).to redirect_to checkout_state_path('address')
       end
     end
-
-    context 'when order is not present' do
```

Comment:
I removed this spec because we don't use the `PUT order/` route to update the cart.